Riot/Categories/MXWellKnown+Riot.swift

```diff
@@ -0,0 +1,62 @@
+// 
+// Copyright 2020 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+extension MXWellKnown {
+    
+    /// Check if a default Jitsi server has been set on the homeserver.
+    ///
+    /// HS admins can set this value in /.well-known/matrix/client:
+    /// "im.vector.riot.jitsi": {
+    ///   "preferredDomain": "your.jitsi.example.org"
+    /// }
+    @objc func vc_jitsiPreferredDomain() -> String? {
```

Comment:
Can we have a URL as returned type?
This conversion can be considered as part of the parsing work.